### PR TITLE
Services version compatibility

### PR DESCRIPTION
**Pull request by Jozefiel**

### CODE REVIEW

File: composefiles/swarm-uniflow.yml [hunk 1]
```diff
@@ -38,7 +38,7 @@ services:
       - PROXY_ENABLED=${PROXY_ENABLED:-false}
       - HTTP_PROXY=${HTTP_PROXY}
       - HTTPS_PROXY=${HTTPS_PROXY}
-      - NO_PROXY=workflow-proxy,frinx-frontend,inventory,${UNICONFIG_ZONES_LIST},unistore,uniresource
+      - NO_PROXY=workflow-proxy,frinx-frontend,inventory,krakend,unistore,uniresource,${UNICONFIG_ZONES_LIST},${NO_PROXY}
```
Comment (SimonMisencik) [inline composefiles/swarm-uniflow.yml:41]: Is ${NO_PROXY} ok? Isn't it recursion?

File: config/krakend/scripts/unistore_rbac.lua [hunk 2]
```diff
@@ -9,40 +17,56 @@ function user_group_auth(request, permited_groups)
...
+                    permited_all=network_role..","..permited_groups
+                end
             end
         end
+    else
+        permited_all=permited_groups..","..service_role..","..network_role..","..bearer_role..","..other_role
     end
 
-    -- remove white spaces and transform to lowercase
+    -- -- remove white spaces and transform to lowercase
```
Comment (SimonMisencik) [inline config/krakend/scripts/unistore_rbac.lua:57]: unnecessary --

File: composefiles/swarm-uniflow.yml (hunk 1 above)
Comment (Jozefiel) [reply composefiles/swarm-uniflow.yml:41]: It's ok. This configures NO_PROXY setting from host + additional settings to docker image.

**Review by SimonMisencik — APPROVED** [review]

File: config/krakend/scripts/unistore_rbac.lua (hunk 2 above)
Comment (Jozefiel) [reply config/krakend/scripts/unistore_rbac.lua]: fixed

Comment (Jozefiel) [reply config/krakend/scripts/unistore_rbac.lua]: fixed

**Pull request merged**